### PR TITLE
docs: ✏️ update Readme to reflect the new changes in the library

### PR DESCRIPTION
**Pull request by NachoVazquez**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Docs are outdated and no longer represent the current implementation.

Issue Number: N/A

## What is the new behavior?

Docs are updated and in sync with `Lumberjack` v2

## Does this PR introduce a breaking change?

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,185 +54,225 @@ Lumberjack is a versatile Angular Logging library, specially defined to be exten
 
 ## Compatibility
 
-`Lumberjack` has tested compatibility with the following Angular versions.
+Lumberjack version 2.x has verified compatibility with the following Angular versions.
 
-> If the version you are using is not listed, please rise an issue. This list only covers tested versions.
+| Angular version | Lumberjack support |
```
Comment (LayZeeDK) [inline README.md:59]:
```suggestion
| Angular version | Lumberjack 2.x support |
```

**Comment by LayZeeDK** [comment]
@NachoVazquez Try to make another commit to force another CI run.

**Review by LayZeeDK — DISMISSED** [review]

**Review by LayZeeDK — APPROVED** [review]

**Comment by LayZeeDK** [comment]
You have this point in a project board item:
>- Extra: Add `LumberjackLevel` design decisions. (diff with classical usage and why)

Probably not something we need right away for the v2.0 release.

**Comment by NachoVazquez** [comment]
> You have this point in a project board item:
> 
> > * Extra: Add `LumberjackLevel` design decisions. (diff with classical usage and why)
> 
> Probably not something we need right away for the v2.0 release.

No, it is just good to have. But it can wait.

**Pull request merged**